lib/vanagon/platform/dsl.rb

```diff
@@ -198,6 +198,20 @@ def build_hosts(*args)
       # Because single vs plural is annoying to remember
       alias_method :build_host, :build_hosts
 
+      # Set a flag whether this platform is cross-compiled
+      #
+      # @param type [Boolean] true if this platform is cross-compiled
+      def cross_compiled(flag)
```

Comment:
These are platform files. But the point stands, and I assume that means you agree with me? we seem to be talking about the same thing